src/main/webapp/edit-recipe.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
Would this page be significantly different (looks wise) from other pages in the project that Angela has coded? If so, I would advise on using a common template for background, headers, footers, logins, etc.

Reply:
I believe that there is an HTML page called create-recipe-form.html that Angela created, but I don't have access to it on this PR because I think it was added to master after this PR was created. Once this is all merged into master, however, I can create a PR that integrates my edit recipe functionality with Angela's HTML.

Reply:
SG.